lib/awful/placement.lua

```diff
@@ -603,6 +617,150 @@ local function rect_to_point(rect, corner_i, corner_j)
     }
 end
 
+-- Create a pair of rectangles used to set the relative areas.
+-- v=vertical, h=horizontal
+local function get_cross_sections(abs_geo, mode)
+print("\n\nLA!",mode)
```

Comment:
The fixup was accidently rebased in the wrong commit. I reviewed the pr as
a whole, so i missed it. Github ui fail quite hard to make that obvious.
On Jun 3, 2016 3:37 AM, "Uli Schlachter" <notifications@github.com> wrote:

In lib/awful/placement.lua
<https://github.com/awesomeWM/awesome/pull/929#discussion_r65667165>:

> @@ -603,6 +617,150 @@ local function rect_to_point(rect, corner_i, corner_j)
>      }
>  end
>
> +-- Create a pair of rectangles used to set the relative areas.
> +-- v=vertical, h=horizontal
> +local function get_cross_sections(abs_geo, mode)
> +print("\n\nLA!",mode)

Uhm. Github GUI says that this is an outdated diff. However, I reviewed
this by git pulling your branch and looking at this locally and noticed the
print there independently from Daniel. What's going on?!?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/awesomeWM/awesome/pull/929/files/405f522b13aa51399e6088c39ecd71086619684f#r65667165>,
or mute the thread
<https://github.com/notifications/unsubscribe/AAUxoNo82wOBni-FNogX9IBBW3F2uQ-Xks5qH9mdgaJpZM4IpNLU>
.